docs/sites.yml

```diff
@@ -3686,3 +3686,17 @@
     - Web Development
     - React-Native
     - Music
+- title: Birra Napoli
+  main_url: http://www.birranapoli.it
+  url: http://www.birranapoli.it
+  featured: false
+  description: >
+    Birra Napoli official site
+  categories:
+    - Landing page
+    - Business 
+    - Technology
+    - Web Development
```

Comment:
Please remove `Technology` and `Web Development` tags, as they do not match the content of the site. See https://www.gatsbyjs.org/docs/site-showcase-submissions/#content-of-site